packages/landing/src/tile.matrix.ts

```diff
@@ -56,7 +56,40 @@ export function locationTransform(
   const coords = projection.fromWgs84([location.lon, location.lat]);
   const center = tileMatrix.sourceToPixels(coords[0], coords[1], Math.round(location.zoom));
   const tile = { x: center.x / tileMatrix.tileSize, y: center.y / tileMatrix.tileSize, z: Math.round(location.zoom) };
-  const mapboxTile = targetTileMatrix.tileToSource(tile);
-  const [lon, lat] = Projection.get(targetTileMatrix).toWgs84([mapboxTile.x, mapboxTile.y]);
+  const wmTile = targetTileMatrix.tileToSource(tile);
+  const [lon, lat] = Projection.get(targetTileMatrix).toWgs84([wmTile.x, wmTile.y]);
   return { lon: Math.round(lon * 1e8) / 1e8, lat: Math.round(lat * 1e8) / 1e8, zoom: location.zoom };
 }
+
+/** Re project WGS84 coordinates into the target tile matrix for use within maplibre */
+export function projectToGoogle(location: [number, number], targetTileMatrix: TileMatrixSet): [number, number] {
```

Comment:
I think we have existing function doing the same thing. `locationTransform()` in `tile.matrix.ts`

Reply:
Hmm when I first tried it it was giving me very weird coordiantes, but looking at it I think If i do `NZTM200QuadTms, GoogleTms` it should maybe work let me try it